Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Setup a Python virtual environment
+FROM python:3.9.13-slim-bullseye as python-builder
+RUN apt-get update && apt upgrade -y && pip install pipenv
+WORKDIR /opt/app
+COPY ./requirements.txt .
+ENV PIPENV_VENV_IN_PROJECT=1
+RUN pipenv install --three
+
+# Build the transifex-cli go static binary
+FROM golang:1-bullseye as go-builder
+WORKDIR /opt/app
+RUN git clone https://github.com/transifex/cli . && make build
+
+# Gather the build artifacts
+FROM python:3.9.13-slim-bullseye
+WORKDIR /opt/app
+COPY . .
+COPY --from=python-builder /opt/app/.venv/ .venv/
+COPY --from=go-builder /opt/app/bin/tx /bin/
+CMD . .venv/bin/activate && mkdocs serve -a 0.0.0.0:8000
```

Comment:
Small styling comment, but newlines at the end of the files are highly appreciated. :)

Reply:
Actually, you can add this as a pre-commit hook. Browse our repos where you can see we use `pre-commit` heavily.
Example configuration about this:

`.pre-commit-config.yml`
```
repos:
  # Fix end of files
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v3.2.0
    hooks:
      - id: trailing-whitespace
      - id: end-of-file-fixer
      - id: mixed-line-ending
        args:
          - '--fix=lf'

```

Reply:
Bonus points: check this via GitHub actions too.

Reply:
Should be enough now. Personally I am not a huge fan of enforcing checks before running something, as you might want to test something quick and dirty, or have to apply immediate change without worrying too much about styling.

Adding a bit of docs how to install pre-commit for local development would be great too.